app/classifier/feedback/feedback-summary-container.jsx

```diff
@@ -35,24 +35,30 @@ class FeedbackSummaryContainer extends React.Component {
     }, []);
     actions.feedback.setFeedback(feedbackItems);
```

Comment:
You're allowed to call static method in the constructors, but I don't think `generateFeedbackItems()` is static. That's why it's complaining about side-effects. I've moved the call out of the constructor and into `componentWillMount()`, which should have the same behaviour.